### PR TITLE
output scroll to bottom implementation

### PR DESCRIPTION
**Pull request by gagandeepp**
Fixes: https://github.com/redhat-developer/openshift-dd-ext/issues/26

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@types/react-test-renderer": "^18.0.0",
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@types/react-window": "^1.8.5",
+    "@types/testing-library__react": "^10.2.0",
```
Comment (fbricon) [inline client/package.json:38]: why do you need this dependency for if you don't add tests?

File: client/src/DeploymentOutput.tsx [hunk 2]
```diff
@@ -7,15 +8,26 @@ interface DeploymentOutputProps {
...
+  const textArea = useRef<HTMLTextAreaElement>(null);
+  
+  // After render, this scrolls the textArea to the bottom.
+  useEffect(() => {
+    if (textArea.current != null) {
+      const area = textArea.current;
+      area.scrollTop = area.scrollHeight;
+    }
+  }),
+    [];
```
Comment (fbricon) [inline client/src/DeploymentOutput.tsx:20]: can you please fix the indentation?

**Review by fbricon — CHANGES_REQUESTED** [review]
Some minor nits, but overall the fix works as expected. Nice work @gagandeepp!

File: client/package.json (hunk 1 above)
Comment (gagandeepp) [reply client/package.json:38]: It was breaking the build

**Comment by gagandeepp** [comment]
@fbricon Can you please review my PR and add **HACKTOBERFEST-ACCEPTED** label to it once you are done reviewing?

File: client/package.json (hunk 1 above)
Comment (fbricon) [reply client/package.json:38]: can you try and remove it? It builds on my machine without it, I wanna check what happens on github actions

Comment (gagandeepp) [reply client/package.json]: @fbricon  Done with the changes ,please check on your side.

File: client/src/DeploymentOutput.tsx [hunk 3]
```diff
@@ -7,15 +8,25 @@ interface DeploymentOutputProps {
...
   const output = props?.deployResponse;
+  const textArea = useRef<HTMLTextAreaElement>(null);
+  
+  // After render, this scrolls the textArea to the bottom.
+  useEffect(() => {
+    if (textArea.current != null) {
+      const area = textArea.current;
+      area.scrollTop = area.scrollHeight;
+    }
+  },[]);
```
Comment (fbricon) [inline client/src/DeploymentOutput.tsx:19]: the empty [] broke the autoscrolling. 

[output] makes it work again

**Review by fbricon — APPROVED** [review]

**Comment by fbricon** [comment]
Thanks for your contribution @gagandeepp!

**Pull request merged**